Tests/Unit/Repository/Resource/PhpcrResourceTest.php

```diff
@@ -20,6 +20,7 @@ class PhpcrResourceTest extends ProphecyTestCase
 
     public function setUp()
     {
+        parent::setUp();
```

Comment:
I will try that later.

Am 19.05.2016 um 13:27 schrieb dantleech:
>
> In Tests/Unit/Repository/Resource/PhpcrResourceTest.php
> <https://github.com/symfony-cmf/resource/pull/15#discussion_r63861865>:
>
> > @@ -20,6 +20,7 @@ class PhpcrResourceTest extends ProphecyTestCase
> >  
> >      public function setUp()
> >      {
> > +        parent::setUp();
>
> We can remove these and the |phpunit-prophecy| dependency. The base
> test case should just extend |PHPUnit_Framework_TestCase|
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/symfony-cmf/resource/pull/15/files/bb855320dfc8f8e54706c37dea6693feb52ce691#r63861865>
>